/**
 * Catch all errors here so we don't have to try / catch in every test and
 * increase verboseness.
 */

process.on('unhandledRejection', (error) => {
  console.log(error)
})
